app/src/main/java/com/example/gabrielsaruhashi/ramp/activities/SubcategoryViewActivity.java

```diff
@@ -103,7 +100,13 @@ public boolean onQueryTextChange(String newText) {
 
     }
 
+    //https://stackoverflow.com/questions/5588804/android-button-setonclicklistener-design
+    public void onClickListen (View view) {
+        //https://stackoverflow.com/questions/6121797/android-how-to-change-layout-on-button-click
```

Comment:
I'd say remove this, click listeners default and have been part of added back in API level 1